src/components/views/rooms/AppsDrawer.js

```diff
@@ -94,15 +94,7 @@ module.exports = React.createClass({
         const hideWidgetKey = this.props.room.roomId + '_hide_widget_drawer';
         switch (action.action) {
             case 'appsDrawer':
-                // When opening the app drawer when there aren't any apps,
-                // auto-launch the integrations manager to skip the awkward
-                // click on "Add widget"
                 if (action.show) {
-                    const apps = this._getApps();
-                    if (apps.length === 0) {
-                        this._launchManageIntegrations();
-                    }
-
```

Comment:
Reason for removing this is that, if there are no widgets, there's now no way to show the apps drawer (there used to be a button that just opened the drawer). This was causing the scalar window to open whenever you pressed 'call' in a group room since it would open the apps drawer but the jitsi widget wouldn't be there yet.